src/transformers/models/t5/modeling_tf_t5.py

```diff
@@ -1156,25 +1131,26 @@ def get_decoder(self):
 
     @add_start_docstrings_to_model_forward(T5_INPUTS_DOCSTRING)
     @replace_return_docstrings(output_type=TFSeq2SeqModelOutput, config_class=_CONFIG_FOR_DOC)
+    @unpack_inputs
```

Comment:
For consistency with the other PRs, it would be nice if you could move `@unpack_inputs` so as to be the top decorator 🙏  (here and in other methods below)

Reply:
Done!